spec/models/manageiq/providers/vmware/cloud_manager/orchestration_stack_spec.rb

```diff
@@ -0,0 +1,56 @@
+describe ManageIQ::Providers::Vmware::CloudManager::OrchestrationStack do
+  let(:ems) { FactoryGirl.create(:ems_vmware_cloud) }
+  let(:orchestration_stack) do
+    FactoryGirl.create(:orchestration_stack_vmware_cloud,
+                       :ext_management_system => ems,
+                       :name                  => 'test',
+                       :ems_ref               => 'one_id')
+  end
+
+  let(:the_raw_stack) do
+    double.tap do |stack|
+      allow(stack).to receive(:id).and_return('one_id')
+      allow(stack).to receive(:human_status).and_return('on')
+    end
+  end
+
+  let(:raw_stacks) do
+    double.tap do |stacks|
+      handle = double
+      allow(handle).to receive(:vapps).and_return(stacks)
+      allow(ems).to receive(:connect).and_return(handle)
+      allow(stacks).to receive(:get_single_vapp).with(orchestration_stack.ems_ref).and_return(the_raw_stack)
+    end
+  end
+
+  before do
```

Comment:
For the current PR this before is reduntnat, however I'd like to continue with other stack operations for which I believe it is nicer to configure ems mock in the before part:

https://github.com/xlab-si/manageiq/pull/2/files#diff-d6014f088e5cc0821687bde7920af7fbR32

Basically, I already have the next patch ready so please just let me know what you prefer: should I fix this one and then change with the next one or is this one ok?